server/app/services/applicant/question/PhoneQuestion.java

```diff
@@ -36,6 +36,19 @@ public ImmutableList<Path> getAllPaths() {
     return ImmutableList.of(getPhoneNumberPath(), getCountryCodePath());
   }
 
+  @Override
+  public ImmutableMap<Path, String> getJsonEntries() {
+    Path path = getPhoneNumberPath().asApplicationPath();
+
+    if (getPhoneNumberValue().isPresent() && getCountryCodeValue().isPresent()) {
+      String formattedPhone =
+          getFormattedPhoneNumber(getPhoneNumberValue().get(), getCountryCodeValue().get());
+      return ImmutableMap.of(path, formattedPhone);
+    } else {
+      return ImmutableMap.of();
+    }
+  }
+
   @Override
```

Comment:
Above code is taken from `case PHONE`:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/export/JsonExporter.java#L130-L219